gitlab/.env-dist

```diff
@@ -0,0 +1,11 @@
+GITLAB_TRAEFIK_HOST=git.example.com
+GITLAB_SSH_PORT=2224
+# Choose Let's Encrypt 'staging' or 'production' environment:
+ACME_CERT_RESOLVER=production
+
+POSTGRES_USER=gitlab
```

Comment:
These should all be namespaced like GITLAB_POSTGRES_USER

(In theory, you should be able to concatenate all of the d.rymcg.tech project .env files together into one and not have any overlaps)